environments/phi-cas/public.yml

```diff
@@ -0,0 +1,118 @@
+DATADOG_ENABLED: True
+DATADOG_INTEGRATIONS_ENABLED: True
```

Comment:
does it need datadog?

Reply:
thought it would be useful to make sure server is performing as expected by visiting DD. Will it create noise? I can remove the slack channel mentioned.

Reply:
ok fine